HelloWorld/HelloWorld/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HelloWorld
+{
+    internal class Program
+    {
+        static void Main(string[] args)
```

Comment:
Если вы не используете дополнительные ключи для приложения, массив аргументов можно не указывать
```suggestion
        static void Main()
```